src/TesApi.Web/Runner/TaskToNodeTaskConverter.cs

```diff
@@ -400,7 +433,6 @@ private static string AppendParentDirectoryIfSet(string inputPath, string pathPa
     /// </summary>
     /// <param name="AdditionalInputs"></param>
     /// <param name="DefaultStorageAccountName"></param>
-    /// <param name="NodeManagedIdentityResourceId"></param>
```

Comment:
Pretty sure this line should be `/// <param name="GlobalManagedIdentity"></param>` rather than removed.